### PR TITLE
More debug commands

### PR DESCRIPTION
**Pull request by obligaron**
visit - let you visit someone in town
exit - exits the game
arrow - change arrow effect

### CODE REVIEW

**Comment by julealgon** [comment]
Would be nice if you could explain what the commands do in more detail in the OP.

**Comment by qndel** [comment]
Actually visiting towners while being in town is quite useless, make it work from dungeon ;)

File: Source/debug.cpp [hunk 1]
```diff
@@ -282,6 +337,35 @@ std::string DebugCmdGenerateItem(const std::string_view parameter)
...
+std::string DebugCmdArrow(const std::string_view parameter)
+{
+	auto &myPlayer = Players[MyPlayerId];
+
+	myPlayer._pIFlags &= ~ISPL_FIRE_ARROWS;
+	myPlayer._pIFlags &= ~ISPL_LIGHT_ARROWS;
+
+	if (parameter == "normal") {
+		// we removed the parameter at the top
+	} else if (parameter == "fire") {
```
Comment (AJenbo) [inline Source/debug.cpp:356]:
```suggestion
	if (parameter == "fire") {
```

File: Source/debug.cpp [hunk 2]
```diff
@@ -182,6 +183,60 @@ std::string DebugCmdLoadMap(const std::string_view parameter)
...
+	{ "pepin", _talker_id::TOWN_HEALER },
+	{ "ogden", _talker_id::TOWN_TAVERN },
+	{ "cain", _talker_id::TOWN_STORY },
+	{ "farnham", _talker_id::TOWN_DRUNK },
+	{ "adria", _talker_id::TOWN_WITCH },
+	{ "gillian", _talker_id::TOWN_BMAID },
+	{ "wirt", _talker_id ::TOWN_PEGBOY },
+	{ "lester", _talker_id ::TOWN_FARMER },
+	{ "girl", _talker_id ::TOWN_GIRL },
+	{ "nuts", _talker_id::TOWN_COWFARM },
```
Comment (AJenbo) [inline Source/debug.cpp:197]:
```suggestion
	{ "nut", _talker_id::TOWN_COWFARM },
```

**Comment by AJenbo** [comment]
> Would be nice if you could explain what the commands do in more detail in the OP.

Not sure what info you are missing, maybe this helps?

visit - teleport to named towner
exit - exits the game
arrow - ported form the original debug code, who knows

**Review by AJenbo — APPROVED** [review]

**Pull request merged**